api/prisma/seed-staging.ts

```diff
@@ -92,6 +93,9 @@ export const stagingSeed = async (
   await prismaClient.translations.create({
     data: translationFactory(jurisdiction.id, jurisdiction.name),
   });
+  await prismaClient.translations.create({
+    data: translationFactory(undefined, undefined, LanguagesEnum.es),
```

Comment:
why no jurisdiction for this translation?

Reply:
no jurisdiction just means that its the generic translation set on the server if that makes sense